client/src/controllers/SlugController.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller } from '@hotwired/stimulus';
+import { cleanForSlug } from '../utils/text';
+
+/**
+ * <input
+ * type="text"
+ * name="slug"
+ * value="slug testing page"
+ * maxlength="255"
+ * aria-describedby="panel-child-promote-child-for_search_engines-child-slug-helptext"
+ * required=""
+ * id="id_slug">
+ */
+export class SlugController extends Controller<HTMLInputElement> {
+  static values = {
+    allowUnicodeSlugs: { default: true, type: Boolean },
```

Comment:
```suggestion
    allowUnicode: { default: true, type: Boolean },
```

Reply:
Just thinking. Maybe the default should be false on the controller. Falsey defaults are usually easier to reason about. 

I know the Settings default is True but that's on server side.